README.md

```diff
@@ -576,6 +585,40 @@ sensu::backend::config_hash:
 
 The first step will not fully add the node to the cluster until the second step is performed.
 
+### Sensu backend federation
+
+Currently the federation support within this module involves configuring Etcd replicators. This allows resources to be sent from one Sensu cluster to another cluster.
+
+It's necessary that Etcd be listening on an interface that can be accessed by other Sensu backends.
+
+First configure backend Etcd to listen on an interface besides localhost:
+
+```puppet
```

Comment:
do we have multiple roles for the sensu backends now such that one is where writes to etcd happen and the other nodes read changes or do all nodes support writes and then sync those between themselves?

Reply:
Not sure I follow the question. Based on Sensu Go docs it's designed so you can setup RBAC resources on one cluster and replicate those RBAC resources to another Sensu Go cluster (roles, role bindings, etc).

Reply:
All resources or specific ones?

Curious how replication works and if there is a master in the cluster or if all nodes are equal.

Reply:
I don't know for sure, just docs reference RBAC resources but don't mention any limits. I think it's just sending resources from one backend to another when they are not in the same cluster.  I tried to make this PR just a tool to manage the replicator and leave the details and explanations of usage be covered in Sensu Go docs, so far all I've tested is sending RBAC resources per Sensu Go docs. The examples I'm providing and using are all from Sensu Go docs, didn't want to try and deviate too much from that.